### PR TITLE
Label

### PR DESCRIPTION
**Pull request by Swarzstein**
## In this pull request:
- Create a Book class in a separate .rb file.
- Create a Label class with an association to the Item class (in a separate .rb file).
- All Book class properties visible in the diagram should be defined and set up in the constructor method.
- All Label class properties visible in the diagram should be defined and set up in the constructor method.
- Implement methods:
  - add_item method in the Label class
    - should take an instance of the Item class as an input
    - should add the input item to the collection of items
    - should add self as a property of the item object (by using the correct setter from the item object)
  - can_be_archived?() in the Book class
    - should override the method from the parent class
    - should return true if parent's method returns true OR if cover_state equals to "bad"
    - otherwise, it should return false
- Add unit tests for all implemented methods.
- Item issues fixed

### CODE REVIEW

**Review by ezraermy — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Swarzstein , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Descriptive pull request. ✔

## Optional suggestions

Please consider the following changes below!👏👏👏

Add missing require statements in 'label_spec.rb': Ensure that all required dependencies are properly imported. In this case, you need to include require 'date' to use the Date class.

**Review by Muhammad0602 — APPROVED** [review]
> ## Status: Approved ✔️
## Hi @Swarzstein 
> 
>Your project looks great; you have done it exceptionally well. 
> 
> ### Congratulations! 🎉 🎉 🎉 🎉
>
ThumbsUpOkayGIF

 
> 
> You may close this Pull Request and merge the changes Keep being awesome devs and good luck with your next milestone!
> 
> ### To Highlight
> ✅ Descriptive pull request message
   ✅ linter error checks validated 
   ✅ Documented your work professionally 
> 
> ### Optional suggestions
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

> Cheers and Happy coding! 👏 👏 👏

**Comment by Muhammad0602** [comment]
I think we should discuss the item.rb together and decide how it should look.

File: classes/label.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  def initialize(title, color)
+    @id = rand(10_000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```
Comment (Muhammad0602) [inline classes/label.rb:16]: As it is stated in the requirements `should add self as a property of the item object (by using the correct setter from the item object)` you should add self as a property in your add_item method.

**Review by Muhammad0602 — CHANGES_REQUESTED** [review]

**Pull request merged**